src/extra/libs/rlottie/lv_rlottie.c

```diff
@@ -122,6 +140,9 @@ static void lv_rlottie_constructor(const lv_obj_class_t * class_p, lv_obj_t * ob
 
     lv_img_set_src(obj, &rlottie->imgdsc);
 
+    rlottie->play_ctrl = LV_RLOTTIE_CTRL_FORWARD | LV_RLOTTIE_CTRL_PLAY | LV_RLOTTIE_CTRL_LOOP;
+    rlottie->dest_frame = (size_t) -1; /* invalid destination frame so it's possible to pause on frame 0 */
```

Comment:
Are you sure it's valid for all compilers? I think an `INT_MAX` constant would be safer.

Reply:
Should be better now.